guides/v2.2/extension-dev-guide/plugins.md

```diff
@@ -288,23 +288,23 @@ class MyUtilityUpdater
 
 ### Prioritizing plugins
 
-The `sortOrder` property for plugins determine when their before, after, or around methods get called when more than one plugins are observing the same method.
+The `sortOrder` property for plugins determines when to call the `before`, `around`, or `after` methods when more than one plugin is observing the same method.
```

Comment:
The before and after methods formatting is inconsistent with the changes above.

Reply:
I changed all `before`, `after`, and `around` formatting to inline code for consistency.